app/models/miq_expression/expression/base.rb

```diff
@@ -0,0 +1,3 @@
+class MiqExpression::Expression::Base
```

Comment:
Should we name this differently.  We already have MiqExpression serialized objects all over the place.  I will forever forget the difference between MiqExpression and Expression::Contains.  Maybe SubExpression or components or something... naming... 😕